### PR TITLE
docs: technical documentation

### LINKED ISSUE #10: Documentation / Diagrams
After research lets create some documentation describing the solution we want to build

### PR DESCRIPTION
**Pull request by JustinCast**
### What does this PR do?

- closes #10 

### Steps to test
Nothing to test

#### CheckList

- [X] Follow proper Markdown format
- [X] The content is adequate
- [] The content is available in both english and spanish
- [X] I Ran a spell check

### CODE REVIEW

**Comment by xavier506** [comment]
@JustinCast  do you have the draw.io source file for `infra.svg` ?

**Comment by JustinCast** [comment]
@xavier506, I don't see that option
image

File: docs/gGoods Infra diagram [hunk 1]
```diff
@@ -0,0 +1 @@
...hOX6dz9DZCEoHekeQMFYEdtPUarKbvDd9BuN5BAntxM78ZRJKOV5E+fTpNb2H/6Ba5xW1pmhmjomj+E28SmiIOWJ9ppr697eamjd09O9vat61xame5unt3dFEZ0mqpplEm5lAkQz2hqvH4/Y38U3UldStCsyOVPpU5nm4bVObtov9p7pwtK1dOD6Rzbg9imDoxNTDGSjEyAxmq3ph/OyX2mrYpl5GL5Nbe7ZI+l/X0azU7aul39dcukt3jsZAVQa81hYlwIwDqOxNHWwltJ4jclfQShuV1e4lDAkmAroMayLtufBsvvDL9AWWGZb59GJw71984stMxAPwQneYGog2YBkrF0q7bNIxA4jV1ssWX7By7ZnSmaDFV5tOoTuLVgenU7R8DIo1eoyzldkd5fQkxOmjinGsqjV7lCrQKvVrzZIEPiUweHyCuvApKKV8oaUUQrsLSIRIc3GEhRG0m5ydL0LJo9RWVLnSSaIJQ4O6ekk2BbN/efjozcarIl56RP7cBEM3GY20C3t6Bu8n8sNk9t3uzjXTHjyYf9yhfDHRrGfj2ZjGY/UyDUBpSNl2UUhLCGkDIL1Vm9rPNsVrW5eVNlEhmbF8PbAUvE2hhihTZl0rWZSgF+1e0Q/QTirevCYAEvCpmI/IgUsPvlLZnvkwRMTVow8zSFsspbslWfsgH05+vbiiuZpjz8hgBw5LFQ4LPBJZ1PBZm6rgG4Zv0RjRfm0qLtJJnp25RasAocF6/6/IJFQRINyhVlf9RYiAv+TNEx+Y1t49if0RFZRN4l13rsGUX82P7pxfqGzSVjRwnPyCcP3MxgoHrebv9GKW427c3pqfe8NW7v1lZtOG7OZ2om6a9X8qwyVtzXCVNpZlqKNZLurnDi6Rj8OPy3AVrbFNyHCpGmD9SUGPFzB5ZwIcYPOQOGDgI4ZzhGPqSv6Evo+oom/6JqJXNjLU2AxtVaICwS6w2nzCLxfRIjx3bsD84dfldKDOr66fBAmEfFPRS8rFOGUdjqh8N5aPf0U8oErVV88BnvdaTckjYUlF9JaDSsx/BmceGXLbvxuW5PmvjPS5lzDVHf9pbBOmLuq/k626Ksk312HwHDw9LR4ewllP64VWGAukfZ32XaXYmxP1b31vwabO+gO+APUV8RXY30K7l3m5x/tQaT6zeOntKr5ZvzpY7f8L</diagram></mxfile>
```
Comment (xavier506) [inline docs/gGoods Infra diagram:1]: @JustinCast  let's rename this file

**Comment by xavier506** [comment]
> @xavier506, I don't see that option

Export as an XML file

**Review by xavier506 — APPROVED** [review]

**Pull request merged**